src/core/style/layout.scss

```diff
@@ -69,6 +74,11 @@
 @mixin layout-for-name($name) {
   [layout-#{$name}] {
     box-sizing: border-box;
+    display: -webkit-box;
```

Comment:
@ThomasBurleson @rschmukler pointed out that autoprefixer should take care of adding prefixed `flex` styles. So do you think this is necessary? It was a fix for #1561, but I'm now not sure if that was even an issue.